plugins/kubernetes-common/report.api.md

```diff
@@ -432,6 +432,10 @@ export interface ResourceRef {
   namespace: string;
 }
 
+// @public
+export const SERVICEACCOUNT_CA_PATH =
+  '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt';
```

Comment:
Maybe I should split this into two constants? One for the path and one for the file name so that I can use the path and name when I mock them. Thoughts?